packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -98,17 +98,31 @@ export class ColorPicker
    * When `true`, an empty color (`null`) will be allowed as a `value`.
    *
    * When `false`, a color value is enforced, and clearing the input or blurring will restore the last valid `value`.
+   *
+   * @deprecated use `clearable` instead
```

Comment:
Can you capitalize `use`? I'll submit a separate PR to ensure we capitalize deprecation messages.